src/qumin/entropy/distribution.py

```diff
@@ -135,6 +204,16 @@ def add_features(self, group):
         else:
             return group.applicable
 
+    def concat_data(self, *args, **kwargs):
```

Comment:
how about `def add_measure()` ? Can we constrain it a little bit more ? Otherwise just keep a simple call to pd.concat ?